elifepubmed/generate.py

```diff
@@ -601,6 +601,45 @@ def set_grants(parent, poa_article):
                 set_object(parent, "grant", params)
 
 
+def dataset_assigning_authority(assigning_authority, uri):
```

Comment:
It's good practice to name functions describing what they do. In this instance `get_dataset_assigning_authority` would suggest it gets an assigning authority from a dataset.